### DISCUSSION TITLE
Can I add models from internet after downloading it?

### DISCUSSION THREAD

**Discussion by Ubuntuuuuu** [carson-katri/dream-textures#580] [Q&A]
I can search some models, but how about add models from Internet-site, like:

https://civitai.com/models/4617/rinamix2

Is it to possible add those somehow? Thanks for help!

**Comment by carson-katri** [comment]
You can import a .ckpt file for any model not available on Hugging Face, as described here: https://github.com/carson-katri/dream-textures/blob/main/docs/SETUP.md#importing-checkpoints
